kube_log_watcher/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '0.34'
+__version__ = '0.36'
```

Comment:
Why `0.36`? Shouldn't `0.35` be the next version?

Reply:
Because I've forgot to bump the version the last time (removing the empty newlines, see changelog))